airbyte-integrations/bases/base-java/src/test/java/io/airbyte/integrations/destination_async/RunningFlushWorkersTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination_async;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import io.airbyte.protocol.models.v0.StreamDescriptor;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RunningFlushWorkersTest {
```

Comment:
chat gpt wrote this entirely! only change i made was to move the descriptors into a constant. 🙊